node/src/Pos/Wallet/Web/State/Acidic.hs

```diff
@@ -143,4 +140,5 @@ makeAcidic ''WalletStorage
     , 'WS.flushWalletStorage
     , 'WS.getWalletStorage
     , 'WST.createAccountWithAddress
+    , 'WST.deleteWallet
```

Comment:
I think this name is confusing. To avoid name clashes in the transaction names we can use numbers. For example we can call it `removeWallet2` and the `removeWallet` wrapper can call `A.RemoveWallet2`.